src/main/java/persistence/sql/ddl/QueryBuilder.java

```diff
@@ -0,0 +1,88 @@
+package persistence.sql.ddl;
+
+import jakarta.persistence.*;
+import org.jetbrains.annotations.NotNull;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+public class QueryBuilder {
+    Class<?> clazz;
+    SqlTypeMapper sqlTypeMapper;
+
+    public QueryBuilder(Class<?> clazz, SqlTypeMapper sqlTypeMapper) {
+        this.clazz = clazz;
+        this.sqlTypeMapper = sqlTypeMapper;
+    }
+
+    public String builder() {
+        return "CREATE TABLE " + getTableName() + "(" + getPersonMetadata() + ")";
+    }
+
+    public String dropper() {
+        return "DROP TABLE " + getTableName() + " IF EXISTS";
+    }
+
+    @NotNull
+    private String getTableName() {
+        if (clazz.isAnnotationPresent(Table.class)) {
+            return clazz.getAnnotation(Table.class).name();
+        }
+        return clazz.getSimpleName();
+    }
+
+
+    public String getPersonMetadata() {
+        StringBuilder query = new StringBuilder();
+        List<String> columnArrayList = new ArrayList<>();
+        for (Field field : clazz.getDeclaredFields()) {
+            if (field.isAnnotationPresent(Transient.class)) continue;
+
+            String column = getColumnName(field) + " " + getSqlType(field) + getPrimaryKey(field) + getAutoIncrement(field) + getNullable(field);
+            columnArrayList.add(column);
+        }
+        query.append(String.join(", ", columnArrayList));
+
+        return query.toString();
```

Comment:
자바의 stream을 사용해서 표현해보면 아래와 같이 구현 해볼 수도 있습니다! 😄 

```suggestion
        return Arrays.stream(clazz.getDeclaredFields())
                .filter(field -> !field.isAnnotationPresent(Transient.class))
                .map(field -> getColumnName(field) + " " + getSqlType(field) + getPrimaryKey(field) + getAutoIncrement(field) + getNullable(field))
                .collect(Collectors.joining(", "));
```